CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# Contributing to WooCommerce Admin
+
+Hi! Thank you for your interest in contributing to WooCommerce Admin. We appreciate it.
+
+There are many ways to contribute – reporting bugs, feature suggestions, and fixing bugs.
+
+## Reporting Bugs, Asking Questions, Sending Suggestions
+
+Open [a GitHub issue](https://github.com/woocommerce/woocommerce-admin/issues/), that's all. If you want to prefix the title with a “Question:”, “Bug:”, or the general area of the application, that would be helpful but is no means mandatory. If you have write access, add the appropriate labels.
+
+If you're filing a bug, specific steps to reproduce are helpful. Please include what you expected to see and what happened instead.
+
+## We're Here To Help
+
+We encourage you to ask for help. We want your first experience with WooCommerce Admin to be a good one, so don't be shy. If you're wondering why something is the way it is, or how a decision was made, you can tag issues with [Type] Question or prefix them with “Question:”
+
+## License
+
+WooCommerce Admin is licensed under [GNU General Public License v2 (or later)](/LICENSE.md).
+
+All materials contributed should be compatible with the GPLv2. This means that if you own the material, you agree to license it under the GPLv2 license. If you are contributing code that is not your own, such as adding a component from another Open Source project, or adding an `npm` package, you need to make sure you follow these steps:
+
+1. Check that the code has a license. If you can't find one, you can try to contact the original author and get permission to use, or ask them to release under a compatible Open Source license.
+2. Check the license is compatible with [GPLv2](http://www.gnu.org/licenses/license-list.en.html#GPLCompatibleLicenses), note that the Apache 2.0 license is *not* compatible.
+3. Add the code source URL (e.g. a GitHub URL), the files where it's used in the WooCommerce Services and the full license terms to [`CREDITS.md`](/CREDITS.md)
```

Comment:
WooCommerce Services -> wc-admin (or WooCommerce Admin)

Reply:
Fixed in https://github.com/woocommerce/wc-admin/pull/748/commits/8d827876244a21487add21a0e78ef8c879bcd300